bridge_adaptivity/module/views.py

```diff
@@ -745,6 +743,23 @@ def callback_sequence_item_grade(request):
     return HttpResponse(xml, content_type="application/xml")
 
 
+def _check_and_build_web_socket_message(sequence, score):
```

Comment:
I made this function because, the Code Climate alarmed the error related to `Cyclomatic complexity is too high in function callback_sequence_item_grade. (11)`